### PR TITLE
Elhalvers/4828 local timezone

### LINKED ISSUE #4828: Add local timezone to DateTime stamp for Edit User Profile
**What type(s) of user does this feature affect?**
 - volunteers? Yes
 - supervisors? Yes
 - admins? Yes
 - all casa admins? Yes

**Description**
Stakeholders are requesting time information to be added to various time-related attributes such as 'Last logged in', 'Email sent at', etc... Recent merge included the time to the date stamp for these attributes but the time displayed is in UTC.

The issue is to capture and use a local time zone so the user can get more value out of the time stamp without having to calculate for the timezone offset.

**Screenshots of current behavior, if any**  
Screenshot 2023-05-15 at 8 21 02 AM

_Login Details:_  
[Link to QA site](https://casa-qa.herokuapp.com/)  

Login Emails: 
- volunteer1@example.com  view site as a volunteer
- supervisor1@example.com view site as a supervisor
- casa_admin1@example.com view site as an admin
- all_casa_admin1@example.com view site as an all casa admin
  - go to `/all_casa_admins/sign_in`  

password for all users: 12345678  

### Questions? Join Slack!

We highly recommend that you join us in slack https://rubyforgood.herokuapp.com/ #casa channel to ask questions quickly and hear about office hours (currently Tuesday 5-7pm Pacific), stakeholder news, and upcoming new issues.

### PR DESCRIPTION
**Pull request by elhalvers**
### What github issue is this PR for, if any?
Resolves #4828 

### What changed, and why?
I use 2 yarn packages and a concern to capture and store the local browser time zone and make it available in order to convert the default UTC time displayed on the 'edit user_profile' page to the local user's time zone, displaying the converted time zone.

### How will this affect user permissions?
- Volunteer permissions: NA
- Supervisor permissions: NA
- Admin permissions: NA

### How is this tested? (please write tests!) 💖💪
The decorator tests did not need to change. I have not yet created a test for the new concern or a javascript test for my new js file. I can work on these.

### Screenshots please :)
Before:
before_time_zone_fix

After:
after_time_zone_fix

### Feelings gif (optional)
What gif best describes your feeling working on this issue? https://giphy.com/
How to embed:
`Party!`

### Feedback please? (optional)
We are very interested in your feedback! Please give us some :) https://forms.gle/1D5ACNgTs2u9gSdh9

### CODE REVIEW

**Comment by elhalvers** [comment]
@cnorm35

**Comment by elhalvers** [comment]
The 2 failing specs do not seem to be related to the PR. 😊

File: db/schema.rb [hunk 1]
```diff
@@ -552,6 +560,16 @@
     t.index ["reset_password_token"], name: "index_users_on_reset_password_token", unique: true
   end
 
+  create_table "versions", force: :cascade do |t|
```
Comment (compwron) [inline db/schema.rb:563]: What is this table for?

File: db/schema.rb [hunk 2]
```diff
@@ -449,6 +449,14 @@
     t.index ["user_id"], name: "index_preference_sets_on_user_id"
   end
 
+  create_table "preferences", force: :cascade do |t|
```
Comment (compwron) [inline db/schema.rb:452]: What is this table for?

**Review by compwron — APPROVED** [review]
Seems reasonable minus the mess. Let's merge and cleanup in situ

File: db/schema.rb (hunk 1 above)
Comment (elhalvers) [reply db/schema.rb:563]: Hi @compwron It was just there when I pulled the current main branch. Is it not in the current main branch?

**Pull request merged**